roles/core/pxe_stack/templates/menu.ipxe.j2

```diff
@@ -40,6 +40,7 @@ item bootdeployclone      Deploy already made clone
 #item memtest              Memtest (To check Memory)
 item --gap                Other
 item startshell           iPXE shell (Go to an iPXE shell)
+item startgrubshell       Grub2 shell (Go to a Grub2 shell)
```

Comment:
Can you add information this is for EFI only?

Reply:
Doing it right now. It is indeed EFI only.